lib/cocoapods/installer/xcode/pods_project_generator/app_host_installer.rb

```diff
@@ -0,0 +1,134 @@
+module Pod
+  class Installer
+    class Xcode
+      class PodsProjectGenerator
+        # Installs an app host target to a given project.
+        #
+        class AppHostInstaller
+          # @return [Sandbox] sandbox
+          #         The sandbox where the support files should be generated.
```

Comment:
sorry this was my bad to tell you to copy paste it but you can make it `The sandbox used for this installation.`